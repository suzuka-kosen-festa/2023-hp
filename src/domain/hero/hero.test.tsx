import { composeStory } from "@storybook/react";
import { render } from "@testing-library/react";
import Meta, { Default, Soon } from "./hero.stories";

describe("[domain/hero] hero.tsx", () => {
  it("should render the default Hero component", () => {
    const Composed = composeStory(Default, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
  it("should render the soon Hero component", () => {
    const Composed = composeStory(Soon, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
});
