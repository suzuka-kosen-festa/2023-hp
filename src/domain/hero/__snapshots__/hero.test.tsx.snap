// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[domain/hero] hero.tsx > should render the default Hero component 1`] = `
<div>
  <section
    class="styles_section__q0lepz0"
  >
    <div
      class="styles_left__q0lepz1"
    >
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_1__q0lepz5"
        src="/src/assets/icons/cloud-1.svg"
      />
    </div>
    <div
      class="styles_center__q0lepz2"
    >
      <img
        alt="未視感"
        class="styles_logo__q0lepza"
        src="/src/assets/icons/logo-static.svg"
      />
    </div>
    <div
      class="styles_right__q0lepz3"
    >
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_2__q0lepz6"
        src="/src/assets/icons/cloud-2.svg"
      />
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_3__q0lepz7"
        src="/src/assets/icons/cloud-3.svg"
      />
    </div>
  </section>
</div>
`;

exports[`[domain/hero] hero.tsx > should render the soon Hero component 1`] = `
<div>
  <section
    class="styles_section__q0lepz0"
  >
    <div
      class="styles_left__q0lepz1"
    >
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_1__q0lepz5"
        src="/src/assets/icons/cloud-1.svg"
      />
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_4__q0lepz8"
        src="/src/assets/icons/cloud-4.svg"
      />
    </div>
    <div
      class="styles_center__q0lepz2"
    >
      <img
        alt="未視感"
        class="styles_logo__q0lepza"
        src="/src/assets/icons/logo-static.svg"
      />
      <h1
        class="styles_title__q0lepzb"
      >
        視たことのない高専祭を。
        <br />
        新しい世界へご招待します。
        <br />
        Coming soon...
      </h1>
    </div>
    <div
      class="styles_right__q0lepz3"
    >
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_2__q0lepz6"
        src="/src/assets/icons/cloud-2.svg"
      />
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_3__q0lepz7"
        src="/src/assets/icons/cloud-3.svg"
      />
      <img
        alt="cloud assets"
        class="styles__q0lepz4 styles_cloud_gridArea_5__q0lepz9"
        src="/src/assets/icons/cloud-5.svg"
      />
    </div>
  </section>
</div>
`;
